Misc/NEWS.d/next/Library/2020-03-08-11-00-01.bpo-39682.AxXZNz.rst

```diff
@@ -0,0 +1 @@
+Remove undocumented support for *closing* a `pathlib.Path` object via its context manager. The context manager magic methods remain, but they are now a no-op, making `Path` objects immutable.
```

Comment:
Can this be split into multiple lines? The max line width for Misc/NEWS is supposed to be [80 chars](https://devguide.python.org/committing/#what-s-new-and-news-entries):

```suggestion
Remove undocumented support for *closing* a `pathlib.Path` object via its
context manager. The context manager magic methods remain, but they are now a
no-op, making `Path` objects immutable.
```